src/main/java/io/simplelocalize/cli/configuration/ConfigurationLoader.java

```diff
@@ -19,12 +21,13 @@
 
   private Logger log = LoggerFactory.getLogger(ConfigurationLoader.class);
 
-  public Configuration load(String path) {
+  public Configuration load(String path) throws UnsupportedEncodingException {
 
     Path configurationFilePath = Paths.get(path);
     log.info("Using configuration file in path: {}", configurationFilePath);
 
-    File file = configurationFilePath.toFile();
+    File file = new File(URLDecoder.decode(String.valueOf(configurationFilePath.toFile()), "utf-8"));;
```

Comment:
I added this because when a path included a special character like `@` it was failing to load that path, since it was being translated to `%40`. My userpath on my machine is an email address, so I kept getting `Could not load file at /Users/marques%40blahblah....`.